interface/src/app/node/node.js

```diff
@@ -299,4 +307,123 @@ angular.module('adage.node', [
       };
     }
   };
-}]);
+}])
+
+.directive('enrichedGenesets', ['MathFuncts', 'PickledGenesetsService', '$log',
+  function(MathFuncts, PickledGenesetsService, $log) {
+    return {
+      templateUrl: 'node/enriched_genesets.tpl.html',
+      restrict: 'E',
+      scope: {
+        organism: '=',
+        hWGenes: '=' // hWGenes is an array of high-weight Genes
```

Comment:
Hmm, Im pretty sure that ``'@'`` wouldn't work here because it would expect hWGenes to be a string, but we are expecting hWGenes to always be a JSON object.